packages/entity-server/src/routes/requests.ts

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
The grouping of `requests` and `responses` might make more sense when there are more, and it's more obvious what the shared elements are, but I would have instinctively grouped them together, and put that in a file alongside the specific route they related to. So you'd have a directory structure more like `routes/FetchEntity/FetchEntityRoute.ts`, `routes/FetchEntity/interfaces.ts`. What do you think about that? You've probably worked through other routes and the way you've got it comes out better?

Reply:
Tbh most of the design for this I put in place assuming that we'd have a few different types of routes/requests (as we had originally planned with the `/:hierarchyCode/ancestors` and `/:hierarchyCode/descendants` endpoints).

Now that we've consolidated our endpoint design into a single structure (which I love btw) I might refactor all this to be a bit more straightforward. I think we still want a different class for each route, so I'll probs go with either:
defining them inline in `routes/FetchEntity/FetchEntityRoute.ts` (renamed to `routes/hierarchy/SingleEntityRoute.ts`)
defining them in a `types.ts` file under `routes/hierarchy/`